src/chapters/exceptions/introduction.rst

```diff
@@ -0,0 +1,161 @@
+.. index:: ! exception, ! throw
+
+Exceptions
+==========
+
+Like most programming languages, C# includes exceptions and exception handling tooling. Some exception objects
+are provided by .NET itself, while others we may encounter from external C# libraries, or write ourselves.
+
+**Exceptions** in C# are objects derived from the ``System.Exception`` class. They are used in circumstances where something
+takes place outside of the ordinary flow of a program. When you want to explicitly call an exception in your code, you do so 
+with the **throw** keyword. We've actually done this before in this book and will return to that example shortly.
+
+Maybe you see the term "exception" and instinctively groan or roll your eyes? Isn't programming only enjoyable when
+it works? Well, yes it is nice when your code performs as you expect and want it to. And in fact, exceptions are a 
+vital component of a healthy codebase. As a programmer, encountering an exception can provide crucial information 
+about proper usage of the. As a program writer, providing exceptions gives fellow programmers necessary intel on how your 
+app should run and be used.
+
+When Exceptions Arise
+---------------------
+
+You may have encountered exceptions in C# or another programming language already. Perhaps in on of these scenarios
+where exceptions often arise:
```

Comment:
You could replace the final sentence with "... already, perhaps in one of the following scenarios: "